pkg/operator/ceph/controller/spec.go

```diff
@@ -57,6 +57,8 @@ const (
 	CephUserID                              = 167
 	livenessProbeTimeoutSeconds       int32 = 5
 	livenessProbeInitialDelaySeconds  int32 = 10
+	readinessProbePeriod              int32 = 5
```

Comment:
```suggestion
	readinessProbePeriodSeconds              int32 = 5
```
?

Reply:
small nits, @rkachach ^^

Reply:
@subhamkrai thanks, actually it's a good suggestion so I'll change it :)

Reply:
Fixed